client/verta/verta/tracking/entities/_deployable_entity.py

```diff
@@ -88,6 +90,77 @@ def _get_artifact_msg(self, key):
     def _get_artifact(self, key):
         raise NotImplementedError
 
+    def log_schema(self, input: dict, output: Optional[dict] = None) -> None:
+        """
+        Sets the input and output schemas of this deployable entity. Schemas are stored as
+        model artifacts.
+
+        The output schema is optional.
+
+        To validate a prediction's input and output against these schemas, use the
+        :func:`~verta.registry.validate_schema` decorator on your model's predict function.
```

Comment:
Is there a way to link to the model's predict function docs? Not sure what's best to link to/reference, here.

Reply:
We could link to

```suggestion
        :func:`~verta.registry.validate_schema` decorator on your model's :meth:`~verta.registry.VertaModelBase.predict` function.
```